WordPress/src/main/res/values/strings.xml

```diff
@@ -263,6 +263,7 @@
     <!-- post view -->
     <string name="post_id">Post</string>
     <string name="upload_failed">Upload failed</string>
+    <string name="draft_uploaded">Draft uploaded</string>
```

Comment:
I think it shouldn't be much of a big deal:  I also considered using `Draft saved online` but it's exactly the same thing that is shown in the snackbar when you exit the editor, so it looked too redundant. For te purpose of the notifications themselves, it seemed clearer and inline with them that `<object>` has been  `<acted upon>` (i.e. draft updated, draft uploaded, post updated, post published, page updated, page published).
Still think this needs more thought?